app/views/applications/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<h1> Application Details </h1>
+
+
+<% if @application.application_status != "Pending" %>
+  <p><strong>Add a Pet to this Application</strong></p>
+  <%= form_with url: "/applications/#{@application.id}/search_pets", method: :get do |form| %>
```

Comment:
This form path could still be the `show` action but the search term would be passed as a parameter